lib/src/package_analyzer.dart

```diff
@@ -38,6 +39,7 @@ class InspectOptions {
     this.isInternal = false,
     this.lineLength,
     this.analysisOptionsUri,
+    this.nullSafetySectionEnabledBeforeOptin = true,
```

Comment:
Shouldn't it default to `false` for now? or are we setting that in `pub.dev`... perhaps it's better to not give it a default value?

Reply:
I'm ok with false.

Done.